### PR TITLE
fixes #4333 - added multi-select gem

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -366,3 +367,55 @@ $(this).removeClass('hidden').removeClass('hide')
 return originalShowMethod.apply( this, arguments );
 }
 })();
+
+
+$(function(){
+  multiSelectOnLoad()
```
Comment (ohadlevy) [inline app/assets/javascripts/application.js]: should this be extracted to a separate file?

**Comment by isratrade** [comment]
rebased with js code is separate file and not application.js

**Comment by abenari** [comment]
looks good to me.
I like the fact that this patch separate the vendor code from our wrapper code.

**Comment by isratrade** [comment]
only failure seems to be jenkins-related

**Comment by domcleal** [comment]
The failure's because the RPM spec file needs updating and this gem needs to be built.

**Comment by domcleal** [comment]
The multi-select gem looks to be outdated, is this a good idea?  It's currently packaging multi-select 0.8, while the latest is 0.9.11.

**Comment by isratrade** [comment]
gem is "multi-select-rails", "~> 0.9.10"
https://github.com/klenis/multi-select-rails
last updated 10 days ago

**Comment by domcleal** [comment]
Ahh, thanks.  I looked up the name in the PR title :)

My only concern is that we use jquery-rails 2.0.x (JQuery 1.7.2 I think) and the current (0.9.x) versions of multi-select-rails say the minimum JQuery version is 1.8.  I know JQuery versions themselves are another can of worms that @abenari's tackling, but are we sure this is compatible with 1.7.2, or should we pin to an older version?

**Comment by isratrade** [comment]
@domcleal, I was also concerned that the gem would work with Jquery 1.7.2 since it says 1.8+, but it seems to be working ok.

**Comment by domcleal** [comment]
Thanks @isratrade, merged as 8285778 for Foreman 1.5.

**Pull request closed without merging**